test/Serilog.PerformanceTests/AllocationsBenchmark.cs

```diff
@@ -101,5 +170,96 @@ public void LogAnonymous()
         {
             _logger.Information("Template: {@AnonymousObject}.", _anonymousObject);
         }
+
+
+        [Benchmark]
+        public void LogMix2()
+        {
+            _logger.Information("Template: {Value1},{Value2}", "42", 7);
+        }
+        [Benchmark]
+        public void LogMix3()
+        {
+            _logger.Information("Template: {Value1},{Value2},{Value3}", "42", 7, 108L);
+        }
+        [Benchmark]
+        public void LogMix4()
+        {
+            _logger.Information("Template: {Value1},{Value2},{Value3},{Value4}", "42", 7, 108L, 1024M);
+        }
+        [Benchmark]
+        public void LogMix5()
+        {
+            _logger.Information("Template: {Value1},{Value2},{Value3},{Value4},{Value5}", "42", 7, 108L, 1024M, (short?)-11);
+        }
+        [Benchmark]
+        public void LogMixMany()
+        {
+            _logger.Information("Template: {Value1},{Value2},{Value3},{Value4},{Value5},{Value6},{Value7},{Value8},{@Value9}", "42", 7, 108L, 1024M, (short?)-11, _bigStruct, _dictionaryValue, _sequence, _anonymousObject);
+        }
+
+        [Benchmark]
+        public bool LogAll()
```

Comment:
Will this one uncover anything the other added benchmarks won't? At first glance it seems like it may be difficult to pinpoint the causes of any regressions in this benchmark.

Reply:
No, this will probably not uncover anything new. Except if is something related to __enrichedLogger_.

My idea with this test was just to have an overview of what changes are making more difference for many different types of calling the Serilog.

Did you prefer me to remove this test?